app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,23 @@
+class SessionsController < ApplicationController
+  before_action :require_logout, except: [:logout]
+
+  def login
+    render :sign_in
+  end
+
+  def create
+    user = User.find_by(email: params[:email])
+    if user && user.authenticate(params[:password])
+      session[:user_id] = user.id
+      redirect_to home_path, notice: "Welcome, #{user.full_name}!"
```

Comment:
Bootstrap 3 (used in myflix) uses `success` and `danger` classes instead of `notice` and `error`.